### ISSUE TITLE
Submission errors unclear

### ISSUE THREAD

**Issue by iadawn** [w3c/w3c-website#226]
[Account creation](https://beta.w3.org/account/request/)

When creating anaccount there are no actual error messages. The 'continue' button remains disabled until all errors are fixed. Where an error might occur is not clear.

**Expected behavior**

Do not disable the submit button.
Where a field (or field combination, for example password confirmation) is in error include:

* A clear indication that the field is in error visually and programmatically - something in the `<label>`
* A clear description of the error and how to fix it

On activating the submit button when the form is in error, move focus to the first field in error. This is not necessarily ideal for larger forms where there are many fields but for a small form like this it should be ok.

**Comment by vivienlacourba** [comment]
> Do not disable the submit button.
> Where a field (or field combination, for example password confirmation) is in error include:
> 
>  * A clear indication that the field is in error visually and programmatically - something in the <label>
>  * A clear description of the error and how to fix it

This part was fixed by @deniak.

> On activating the submit button when the form is in error, move focus to the first field in error. This is not necessarily ideal for larger forms where there are many fields but for a small form like this it should be ok.

I guess this is a common issue for all our forms and might be trickier to fix.

**Comment by iadawn** [comment]
The focus management only really becomes a thing if the approach is to do client-side error handling. The approach in the [Beta Authorisation Portal](https://beta.w3.org/account/request/) uses server-side error handling which means that focus control isn't necessary.

Happy to close this issue if the 'do not disable the submit button' has been actioned.
